cms/templates/widgets/footer.html

```diff
@@ -1,14 +1,16 @@
 <%!
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
+from datetime import datetime
+import pytz
 %>
 
 <div class="wrapper-footer wrapper">
   <footer class="primary" role="contentinfo">
 
     <div class="footer-content-primary">
       <div class="colophon">
-        <p>&copy; ${settings.COPYRIGHT_YEAR} <a data-rel="edx.org" href="${marketing_link('ROOT')}" rel="external">${settings.PLATFORM_NAME}</a>.</p>
+        <p>&copy; ${datetime.now(pytz.UTC).year} <a data-rel="edx.org" href="${marketing_link('ROOT')}" rel="external">${settings.PLATFORM_NAME}</a>.</p>
```

Comment:
@Salomari1987 I think it's a good idea to use this context processor variable instead of hardcoding the time zone.